app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+class UsersController < ApplicationController
+  include SessionsHelper
+  include UsersHelper
+
+  def index
+    @users = User.all
+  end
+
+  def show
+    if current_user.nil?
+      redirect_to login_path
+    else
+      @user = User.find(params[:id])
+    end
+  end
```

Comment:
This method should include these variables to render in the views (remember logic shouldn't be in the views):
- the list of events created by the user
- list of events previously attended by the user, using the event model scope. See [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png).
- list of upcoming events previously attended by the user, using the event model scope. See [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png).